pysmiles/smiles_helper.py

```diff
@@ -288,8 +295,31 @@ def remove_explicit_hydrogens(mol):
                     mol.edges[n_idx, neighbor].get('order', 1) != 1):
                 # The molecule is H2, or the bond order is not 1.
                 continue
+            if mol.nodes[n_idx].get('ez_isomer'):
+                anchor = mol.nodes[n_idx]['ez_isomer'][1]
+                for ez_neighbor in mol[anchor]:
+                    if (ez_neighbor != n_idx
+                            and mol.edges[anchor, ez_neighbor].get('order', 1) == 1
+                            and ez_neighbor not in to_remove):
+                        # Found the other ligand that's connected to the anchor
+                        # with a single bond
+                        mol.nodes[ez_neighbor]['ez_isomer'] = mol.nodes[n_idx]['ez_isomer'].copy()
+                        mol.nodes[ez_neighbor]['ez_isomer'][-1] = 'trans' if mol.nodes[ez_neighbor]['ez_isomer'][1] == 'cis' else 'cis'
+                        mol.nodes[ez_neighbor]['ez_isomer'][0] = ez_neighbor
+
+                        ez_partner = mol.nodes[n_idx]['ez_isomer'][3]
+                        mol.nodes[ez_partner]['ez_isomer'][3] = ez_neighbor
+                        mol.nodes[ez_partner]['ez_isomer'][-1] = mol.nodes[ez_neighbor]['ez_isomer'][-1]
```

Comment:
I assume this deals with hydrogen atoms that have a \/ indicator; do you annotate the anchor node?

Reply:
Correct. But the anchors are not annotated anyway.